app/lib/server/functions/relinquishRoomOwnerships.js

```diff
@@ -0,0 +1,81 @@
+import { hasRole, getUsersInRole, addUserRoles } from '../../../authorization';
+import { FileUpload } from '../../../file-upload';
+import { Users, Subscriptions, Messages, Rooms } from '../../../models';
+
+const bulkRoomCleanUp = (rids) => {
+	// no bulk deletion for files
+	rids.forEach((rid) => FileUpload.removeFilesByRoomId(rid));
+
+	return Promise.await(Promise.all([
+		Subscriptions.removeByRoomIds(rids),
+		Messages.removeByRoomIds(rids),
+		Rooms.removeByIds(rids),
+	]));
+};
+
+export const relinquishRoomOwnerships = function(userId, removeDirectMessages = true) {
+	const roomCache = [];
+
+	// Iterate through all the rooms the user is subscribed to, to check if they are the last owner of any of them.
+	Subscriptions.db.findByUserId(userId).forEach((subscription) => {
+		// DMs don't have owners
+		if (subscription.t === 'd') {
+			return;
+		}
+
+		const roomData = {
+			rid: subscription.rid,
+			t: subscription.t,
+			subscribers: null,
+		};
+
+		if (hasRole(userId, 'owner', subscription.rid)) {
+			// Fetch the number of owners
+			const numOwners = getUsersInRole('owner', subscription.rid).fetch().length;
+			// If it's only one, then this user is the only owner.
+			if (numOwners === 1) {
+				// Let's check how many subscribers the room has.
+				const options = { sort: { ts: 1 } };
+				const subscribersCursor = Subscriptions.findByRoomId(subscription.rid, options);
+				roomData.subscribers = subscribersCursor.count();
```

Comment:
this `subscribers` property is not used anywhere else.. is something missing or was the property left by mistake?